system-container/root/exports/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "version": "1.0",
+    "defaultValues": {
+        "OPTS": "",
+        "VAR_LIB_OPENSHIFT_INSTALLER" : "/var/lib/openshift-installer",
+        "VAR_LOG_OPENSHIFT_LOG": "/var/log/ansible.log",
+        "PLAYBOOK_FILE": "/usr/share/ansible/openshift-ansible/playbooks/byo/config.yml",
```

Comment:
> The usage instructions today use relative paths (https://github.com/openshift/openshift-ansible/blob/master/README_CONTAINER_IMAGE.md), wouldn't it work in this case?

took me a while to realize it wouldn't due to the difference: `cwd` in `config.json.template` vs `WORK_DIR` in `Dockerfile.rhel7`